.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: python
+python:
+- '3.6'
+
+before_install:
+- sudo rm -f /etc/boto.cfg
+install:
+- pip install -e git+https://github.com/spulec/moto@master#egg=moto
```

Comment:
why is moto being installed from source and not PyPI?

Reply:
Due to some random error involving compatibility with TravisCI

Reply:
It's an open issue and this is a work around

Reply:
typical. sorry i chose a library that does that